media/Config/buildDom.js

```diff
@@ -0,0 +1,214 @@
+  const h2Tag = document.querySelector('#toolName');
+  h2Tag.innerText = `Options for ${tool.type}`;
+
+  // erase event listeners in useBtn
+  const tmpBtn = document.querySelector('#useBtn');
+  const useBtn = tmpBtn.cloneNode(true);
+  tmpBtn.parentNode.replaceChild(useBtn, tmpBtn);
+
+  // add event listener to useBtn
+  if (tool.type.startsWith('one-import')) {
+    useBtn.addEventListener('click', oneImportToggleFunction);
+  } else {
+    useBtn.addEventListener('click', function() {
+      const optionFieldset = document.querySelector('#options');
+
+      if (tool.use === true) {
+        tool.use = false;
+        optionFieldset.disabled = true;
+      } else {
+        tool.use = true;
+        optionFieldset.disabled = false;
+      }
+
+      autoCompletePath(tool);
+    });
+  }
+  chooseDisable(tool);
+};
+
+// build DOM for selected tool
+// make tags for options
+const buildOptionDom = function(tool) {
+  changeCommonTags(tool);
+
+  const optionsNameTag = document.querySelector('#optionsName');
+  const optionsValueTag = document.querySelector('#optionsValue');
+  const nameUlTag = document.createElement('ul');
+  const valueUlTag = document.createElement('ul');
+
+  for (let i = 0; i < tool.options.length; i++) {
+    const nameLiTag = document.createElement('li');
+    const valueLiTag = document.createElement('li');
+
+    if (tool.options[i].optionType) {
+      // case for select tag
+      nameLiTag.innerText = tool.options[i].optionName;
+
+      const select = makeSelectTag(tool, i);
+      valueLiTag.appendChild(select);
+
+    } else {
+      if (typeof tool.options[i].optionValue === 'boolean') {
+        // case for toggle button
+        nameLiTag.innerText = tool.options[i].optionName;
+
+        const toggleBtn = makeToggleBtn(tool, i);
+        valueLiTag.appendChild(toggleBtn);
+      } else if (typeof tool.options[i].optionValue === 'string') {
+        nameLiTag.innerText = tool.options[i].optionName;
+
+        if (tool.options[i].optionName === 'input_path') {
+          // case for input_path
+          const inputTag = makeInputPathInput(tool, i);
+          valueLiTag.appendChild(inputTag);
+
+          const btnTag = makeInputPathBtn(tool);
+          valueLiTag.appendChild(btnTag);
+        } else if (tool.options[i].optionName === 'output_path') {
+          // case for output_path
+          const inputTag = makeOutputPathInput(tool, i);
+          valueLiTag.appendChild(inputTag);
+
+        } else {
+          // case for normal input tag
+          const inputTag = makeInputTag(tool, i);
+          valueLiTag.appendChild(inputTag);
+        }
+      }
+    }
+    valueUlTag.appendChild(valueLiTag);
+    nameUlTag.appendChild(nameLiTag);
+  }
+  optionsValueTag.appendChild(valueUlTag);
+  optionsNameTag.appendChild(nameUlTag);
+};
+
+// function for selecting framework
+// you can find oneImportBcq, Onnx, Tf, Tflite in tools.js
+// you can find chooseImportOption in buildImportDom.js
```

Comment:
Not related this PR but I'm curious about this. Can't it just be imported from other js having `export` keyword like below?

```js
import oneImportBcq, Onnx, Tf, Tflite from ./tools.js
```

Reply:
this separation of js files is not for importing or exporting each other.
these files will be concatonated in index.html like one js file.
we choose this way because importing or exporting js files didn't work well in vscode environment.

Reply:
FYI, there is no `import` in Browser(or WebView) JavaScript. All JS files loaded by HTML run in a flat environment.
`import` was introduced by node.js, in case of JS world.

Reply:
@OTKRyu @seanshpark Thank you for you information. I think we could find more better approach for loading html within vscode extension someday.